static/js/components/CourseList.js

```diff
@@ -29,37 +43,75 @@ class CourseList extends React.Component {
           coursesPassed++;
         }
 
-        return <ul
+        let courseRuns = [];
+        if (expander[course.id]) {
+          courseRuns = course.runs.map(run => {
+            return <Grid style={{height: DASHBOARD_RUN_HEIGHT}} key={run.id}>
+              <Cell
+                col={6}
+                className="course-run-title"
+              >
+                {run.title}
+              </Cell>
+              <Cell col={4} className="course-run-status">
+                {makeRunStatusDisplay(run)}
+              </Cell>
+            </Grid>;
+          });
+        }
```

Comment:
I think you should be able to do this for the arrow function here:

```js
run => (
  <Grid ...>
    ...
  </Grid>
);
```

since I believe JSX is transformed into a single expression we're ok doing an implicit-return style arrow function (we don't need curly braces around the function body or the `return` keyword).